packages/design-system/src/tokens/ods/size.yaml

```diff
@@ -22,3 +22,8 @@ size:
     value: 22px
   form-check-default:
     value: 14px
+  tiles:
+    resize-step:
```

Comment:
If `resize-step` is the only size value for tiles, we could omit the nesting an just have something like
```yml
  tiles-step:
    value: 12rem
```